packages/dom-adapters/src/BlockToolAdapter/index.ts

```diff
@@ -171,39 +203,83 @@ export class BlockToolAdapter {
       return;
     }
 
-    const action = event.detail.action;
+    if (nativeInput) {
+      const currentElement = input as HTMLInputElement | HTMLTextAreaElement;
+      const [ [start, end] ] = event.detail.index;
+
+      // todo:
+      //  caretAdapter cannot handle native input
+      //  because it doesn't have the content of the input
```

Comment:
Please, use block-style comments and JSDoc-style todos as we use in a remaining code
```suggestion
      /** 
       * @todo caretAdapter cannot handle native because it doesn't have the content of the input
       */
```